src/backend/commands/explain_gp.c

```diff
@@ -1851,6 +1851,66 @@ cdbexplain_showExecStats(struct PlanState *planstate, ExplainState *es)
 	}
 	pfree(extraData.data);
 
+	/*
+	 * Print "Rows out"
+	 */
+
+	if (es->analyze && ns->ninst > 1) {
```

Comment:
Maybe it could be better to print additional info only if special GUC (for example explain_print_rows_out) is set to on? Good for backward compability

Reply:
Added `gp_enable_explain_rows_out` GUC like `gp_enable_explain_allstats`